PlaybackLab/SerializationDownloadsSampleApp/SerializationDownloadsSampleApp/Utils/Queue.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Queue.swift
+//  SerializationDownloadsSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import Foundation
+
+struct Queue<T> {
```

Comment:
I don't think it makes sense to make Queue generic (`<T>`) since you are always using an `OOAssetDownloadManager`. You can either remove it or make the `Queue` really make use of the generic type `T`